docs/5.0/kubernetes-access.md

```diff
@@ -5,61 +5,231 @@ description: How to set up and configure Teleport for Kubernetes access with SSO
+    #...
+    auth_service:
+    # Example Static Kube invite token. Note V1 only supports static
+    # invite tokens.
+      tokens:
+      - kube:866c6c114724a0fa4d4d73216afd99fb1a2d6bfde8e13a19
+    #...
+    proxy_service:
+      public_addr: proxy.example.com:3080
+      kube_listen_addr: 0.0.0.0:3026
+    ```
+
+    Setup Teleport Helm chart repository.
+
+    ```bash
+    helm repo add teleport https://charts.releases.teleport.dev
+    ```
+
+    | Things to set | Description |
+    |-|-|
+    | `proxyAddr` | The Address of the Teleport Root Service, using the proxy listening port |
+    | `authToken` | A static `kube` invite token |
+    | `kubeClusterName` | Kubernetes Cluster name (there is no easy way to automatically detect the name from the environment) |
+
+    Install the Helm Chart.
+
+    ```bash
+    helm install teleport-kube-agent teleport/teleport-kube-agent \
+      --namespace teleport \
+      --create-namespace \
+      --set proxyAddr=proxy.example.com:3080 \
+      --set authToken=$JOIN_TOKEN \
+      --set kubeClusterName=$KUBERNETES_CLUSTER_NAME
+    ```
+
+=== "Using Trusted Cluster: Teleport Enterprise Example"
+
+    Prepare a Teleport Enterprise license file<br>
+    If you are deploying the Enterprise version you will require the license file as a secret available to Teleport.
+    Download the `license.pem` from the [Teleport dashboard,](https://dashboard.gravitational.com/web/login) and rename
+    it to the filename that this chart expects:
+
+    ```bash
+    cp ~/Downloads/license.pem license-enterprise.pem
+    ```
+
+    Store it as a Kubernetes secret:
+    ```
+    kubectl create secret generic license --from-file=license-enterprise.pem
+    ```
+
+    Setup Teleport Helm chart repository.
+    ```bash
+    helm repo add teleport https://charts.releases.teleport.dev
+    ```
+
+    Download Demo Helm Value and Install Chart
+    ```bash
+    curl https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-auto-trustedcluster/values.yaml --output values.yaml
+    helm install teleport teleport/teleport-auto-trustedcluster --values=values.yaml
+    ```
+
+    Edit `values.yaml` defining [Trusted Cluster Configuration](https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-auto-trustedcluster/values.yaml#L41-L55)
+
+    ```bash
+    helm upgrade -f values.yaml teleport teleport/teleport-auto-trustedcluster
+    ```
+
+    Check that the Teleport Service is running.
+    ```bash
+    kubectl describe deployments/teleport
+    ```
+
+    Teleport should now be accessible from your root cluster.
+
+=== "Using kubeconfig on a Teleport root."
+
+    Prior to Teleport 5.0, this was the only way to setup Teleport. This option
+    `kubernetes_service` elsewhere, with kubeconfig. Use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) for building kubeconfigs.
```

Comment:
Needs more explanation here.
Also, some info on how to build kubeconfigs for multiple clusters by hand? May be a good blog post material